proposals/dao/fip_35.ts

```diff
@@ -0,0 +1,66 @@
+import { ethers } from 'hardhat';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import {
+  DeployUpgradeFunc,
+  NamedContracts,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '../../types/types';
+
+chai.use(CBN(ethers.BigNumber));
+
+// Constants
+const TIMED_MINTER_FREQUENCY = '604800'; // weekly
```

Comment:
Period, but w/e

Reply:
Its frequency just do one round of mental gymnastics